qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -60,20 +85,29 @@ def enable_account(
             the legacy Quantum Experience and Qconsole (also known as the
             IBM Quantum Experience v1) credentials is no longer supported.
 
+            If using a default provider, all three parameters `hub`, `group`, `project`
+            must be specified.
```

Comment:
Why do all three parameters need to be specified for `enable_account()`? Since this method loads all the `AccountProvider` instances, it should be able to just return the provider found by `get_provider()` (which doesn't require all three parameters).

Reply:
The idea was to make sure the user does not accidentally receive a provider they didn't exactly specify. But, it does seem a bit limiting, especially since not all the arguments are required by `get_provider`. I'll go ahead and change that.